### DISCUSSION TITLE
Slug fields

### DISCUSSION THREAD

**Discussion by dimkys** [Laravel-Backpack/community-forum#702] [Q&A (Help)] [answered]
Is it possible to use the backpack functionality to turn text in any language into a slug?

**Comment by pxpm** [comment]
Hello @dimkys . 

I wouldn't say it's possible for **every** language without additional work. We had problems previously with hebraic text for example https://github.com/Laravel-Backpack/CRUD/pull/5272

I would say for most of the languages you can use the `slug` field https://backpackforlaravel.com/docs/6.x/crud-columns#slug-pro or create an mutator/acessor with Laravel in your model to do that for you with `Str::slug()`.

Cheers

**Comment by dimkys** [comment]
@pxpm I’m talking about the functionality that, on the front-end side, takes data from the associated field and inserts it into the slug input.
and yes, I just need Hebrew

**Reply by pxpm** [reply, marked as answer]
I think with the package https://github.com/cviebrock/eloquent-sluggable you are able to generate slugs from hebrew. 

From the front-end-side, you can use a script like in our example: https://backpackforlaravel.com/docs/6.x/crud-fields-javascript-api#9-when-a-text-input-is-written-into-write-in-a-second-input-eg

Not sure it will work with hebrew strings, so you may need to change it to fit your usecase.

Cheers

**Reply by dimkys** [reply]
Yes, I did that, redefined slug.blade.php. So that he doesn't cut hebrew.
Thank you.
